srcpkgs/grub/template

```diff
@@ -141,19 +148,31 @@ grub-utils_package() {
 		vmove usr/bin/grub-mkfont
 	}
 }
-grub-x86_64-efi_package() {
+
+supplementary_package() {
```

Comment:
this is unnecessary and isn't even has the required underscore prefix.

Just repeat the definition on each pacakge.